code/modules/hydroponics/grown/tomato.dm

```diff
@@ -147,3 +174,16 @@
 	K.health = K.maxHealth
 	K.visible_message("<span class='notice'>The Killer Tomato growls as it suddenly awakens.</span>")
 	qdel(src)
+
+/obj/item/food/grown/tomato/killer/pickup(mob/user)
+	. = ..()
+	if(!iscarbon(user))
+		return
+
+	var/mob/living/carbon/carbon_user = user
+	if(HAS_TRAIT(carbon_user, TRAIT_PLANT_SAFE))
+		return
+
+	if(prob(25))
+		to_chat(carbon_user, "<span class='danger'>\The [name] begins to growl and shake!</span>")
```

Comment:
```suggestion
		to_chat(carbon_user, "<span class='danger'>[src] begins to growl and shake!</span>")
```

Reply:
![image](https://user-images.githubusercontent.com/51863163/115103571-6e178a00-9f18-11eb-8431-5b761c47ddf5.png)
*shrug

Reply:
Try renaming it to `\improper killer-tomato`.